client/src/modules/stock/entry/modals/lots.modal.js

```diff
@@ -122,18 +122,21 @@ function StockDefineLotsModalController(Instance, Notify, uiGridConstants, Data,
   }
 
   function checkLine(line, date) {
-       
+    var isPosterior;
+
     if (date) { line.expiration_date = date; }
-    
-    var isPosterior = new Date(line.expiration_date) >= new Date();
+
+    isPosterior = new Date(line.expiration_date) >= new Date();
```

Comment:
"Posterior" is often used in English to refer to someone's butt.  A better name would be "isAfter" or "isLater" or something like that.

However, it might even be better to name this based off its function.  It checks if the stock is expired, correct?  So you could name it `hasFutureExpirationDate` or something along those lines.